React/Getting started with React/Ecosystem.md

```diff
@@ -69,3 +72,5 @@ Automated testing is very important in software development. It gives us the ass
 ### Internationalization
 
 We use [polyglot-cli](https://www.npmjs.com/package/polyglot-cli) for managing translations and [react.i18next](https://react.i18next.com/) for implementing internationalization in React applications.
+
+Now that you have everything set and ready, you can start with the tutorials and official documentation explained in the next chapter.
```

Comment:
It would be great to also add links here. The sidebar is great if you know what you're looking for, but in this "storytelling" part, the goal should be to not use the sidebar at all.